### PR TITLE
[KFSQA-1182] Add a default AFT system parameter that can be used to r…

### PR DESCRIPTION
**Pull request by nkimble**
…estrict the number of data rows returned by the web service that are then converted into ruby string arrays/hashes used in the tests.

### CODE REVIEW

**Comment by nkimble** [comment]
kuality-kfs PR#8 and kuality-kfs-cu PR #13 need to be performed together.

Technically these changes should not have been needed but they appear to be adding stability to the tests and reducing the Timeout errors.

File: lib/kuality-kfs/global_config.rb [hunk 1]
```diff
@@ -354,7 +353,11 @@ def find_cg_accounts_in_cg_responsibility_range(lower_limit, upper_limit)
       raise RuntimeError, "No CG Accounts with CG Account Responsibility ID in range #{lower_limit} to #{upper_limit} found."
     elsif accounts_hash.has_key?('org.kuali.kfs.coa.businessobject.Account')  # multiple accounts found
       accounts_array = accounts_hash['org.kuali.kfs.coa.businessobject.Account']
-      accounts_array.each{ |value|
+      # web service can return an overwhelming amount of data, only take up to a max number
+      # value is used more than once so make one call and reuse locally
+      max_rows = (get_aft_parameter_value(ParameterConstants::DEFAULT_MAX_NUM_DATA_ROWS_TO_USE)).to_i
+      accounts_array.length > max_rows ? number_of_elements = max_rows : number_of_elements = accounts_array.length
```
Comment (kyleoliveira) [inline lib/kuality-kfs/global_config.rb]: This should be changed to:

``` ruby
number_of_elements = accounts_array.length > max_rows ? max_rows : accounts_array.length
```

or alternatively

``` ruby
number_of_elements = (accounts_array.length, max_rows).min
```

**Comment by kyleoliveira** [comment]
Just had the one comment. Let me know what you think.

File: lib/kuality-kfs/global_config.rb (hunk 1 above)
Comment (nkimble) [inline lib/kuality-kfs/global_config.rb]: Thanks for the second set of eyes.  Implemented your first suggestion.  changes have been made and are available for you to look at again.

**Pull request merged**